src/main/scala/nl.knaw.dans.easy.bag2deposit/AmdTransformer.scala

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.bag2deposit
+
+import better.files.File
+import nl.knaw.dans.easy.bag2deposit.ddm.Provenance
+import org.apache.commons.csv.CSVFormat.RFC4180
+
+import java.nio.charset.Charset
+import scala.util.Try
+import scala.xml.Node
+import scala.xml.transform.{ RewriteRule, RuleTransformer }
+
+class AmdTransformer(cfgDir: File) {
+  private val csvFile: File = cfgDir / "account-substitutes.csv"
+  private val userMap = if (!csvFile.exists || csvFile.isEmpty)
+                          Map[String,String]()
+                        else parseCsv(
+                          csvFile,
+                          nrOfHeaderLines = 1,
+                          format = RFC4180.withHeader("old", "new"),
+                        ).map(r => r.get("old") -> r.get("new")).toMap
+
+  private val userRewriteRule: RewriteRule = new RewriteRule {
+    override def transform(node: Node): Seq[Node] = {
+      if (!Seq("depositorId", "signerId").contains(node.label)) node
```

Comment:
signerId is from the agreements.xml, not the amd

Reply:
it is slowly coming back now. The transformer can do both. What is actually done is determined at https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/2cc92759b0cd276c8bb96a14680957eecf1bcf54/src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala#L105-L107